### PR TITLE
Add TFP 22-23

### PR DESCRIPTION
**Pull request by jasonhemann**
Adding TFP 22-23, and cleaning up some instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ selector, press the "Display selected names" button.
 II. Adding conferences to PC-miner
 ----------------------------------
 The PC-miner generator tool is a Java application that you can build and run using
-Eclipse. To run PC-miner, right-click on the class pcminer.PCMiner, and select 
+Eclipse. To run PC-miner, right-click on the class pcminer.PCMiner.java, and select
```
Comment (msridhar) [inline README.md:57]: I think we should either leave this as is, or say to right-click on the file `src/pcminer/PCMiner.java`.  `PCMiner.java` is not a class.

File: README.md [hunk 2]
```diff
@@ -130,7 +130,7 @@ Adding a conference involves the following steps:
          
 5. Once your files are all in place, you can generate the UI by running PCMiner.java.
    (To compile and run, you need in your classpath the jar files in subdirectory lib.)
-   This will generate an updated version of the file ui/data.js. Opening
+   This will generate an updated version of the file ui/js/data.js. Opening
```
Comment (msridhar) [inline README.md:133]: 👍 Thanks for catching this!

**Review by msridhar — COMMENTED** [review]
Sorry for the slow review!  This mostly looks good, just one thing to look at

File: README.md (hunk 1 above)
Comment (jasonhemann) [reply README.md]: Reverted in new commit.

**Comment by jasonhemann** [comment]
I was way slower to reply! Corrected, and merged in changes in the meantime.

**Pull request merged**